socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,28 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
+            store_name = getattr(
+                a_store, 'wrapped_object', a_store.__class__
             )
+            # self.logger.debug('writing %s to %s' % (crash_id, store_name))
+
+            self.quit_check()
+            try:
+                a_store.save_raw_and_processed(
+                    raw_crash,
+                    dump,
+                    processed_crash,
+                    crash_id
+                )
+            except Exception:
+                self.logger.error(
+                    '%s failure: %s', store_name, exc_info=True
```

Comment:
Perhaps also include the `crash_id`. 

Also, what you're doing here is printing the `repr` of the crashstorage class. Not the name. The name is `store_name.__name__`. So, if you intended to write the name, use `.__name__` and/or change the name of the variable from `store_name` to something else. 

Or if you really want the `repr` of the module, which I suspect actually carries more information, you should use `%r` to include it in the logging string (and not bother with the `__name__`).